src/components/sortControl/sortControl.tsx

```diff
@@ -0,0 +1,34 @@
+import './sortControl.css';
+
+interface SortControlProps {
+  sortOptions: movieSortOption[];
+  defaultSelection: movieSortOption;
+  onSelect: (selectedOption: movieSortOption) => void;
+}
+
+const SortControl = (props: SortControlProps) => {
```

Comment:
Please explicity destructure props:

1.option (for a few props): const SortControl = ({defaultSelection, onSelect}: SortControlProps) => {
2.option (for numerous props):
    const SortControl = (props: SortControlProps) => {
        const {defaultSelection, onSelect} = props